web_accounts_list.json

```diff
@@ -3524,6 +3524,17 @@
         "known_accounts" : ["DulcieRichard","Selenaneshy"],
         "category" : "XXXPORNXXX",
         "valid" : true
-      }
+      },
+      {
+      "name": "Cults3D",
+      "check_uri": "https://cults3d.com/en/users/{account}/creations",
+      "account_existence_code": "200",
+      "account_existence_string": "The 3D models of",
+      "account_missing_string": "Oh dear, this page is not working! ・ Cults",
+      "account_missing_code": "200",

```

Comment:
The HTTP response code I get with a not found profile is `404`. Can you double check that and change this value if needed?